File: Practices/fizzbuzz.py
#AL, Fizzbuzz
i = 0
while i < 50:
    i += 1
    if   i % 3 == 0 and i % 5 == 0:
        print("Fizzbuzz")
    elif i % 3 == 0: 
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)

File: Practices/fizzbuzz.c
// AL, Fizzbuzz 3
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(){
   for(int i = 0; i <= 50; i++){    
       if (i % 3 == 0 && i % 5 == 0){
            printf("\nFizzbuzz");
    }   else if (i % 3 == 0){
            printf("\nFizz");
    }   else if (i % 5 == 0){
            printf("\nBuzz");
    }   else{
            printf("\n%d", i);
    }
}
    return 0;
}
